.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+php:
+  - 7.1
+
+cache:
+  directories:
+    - ~/.composer/cache
+    - $HOME/.npm
```

Comment:
you use `$HOME` once and then `~`. Why? Arent they same?

Reply:
also, you can cache things just using keyword npm, which will cache `$HOME/.npm` or `node_modules`. Have you considered it?

Reply:
I used `~`. I didn't use keyword npm, because then i could not cache composer.